Makefile

```diff
@@ -100,31 +100,29 @@ toc:
 	echo "* [$$line](#$$anchor)"; \
 	done
 
-.PHONY hooks
+HOOK = .git/hooks/pre-commit
+.PHONY: hooks
 hooks:
```

Comment:
Here starts the `make hook` implementation in the Makefile.
It creates the file `.git/hooks/pre-commit`